windows/T1140.csv

```diff
@@ -0,0 +1,11 @@
+T1140,windows,dc6fe391,Deobfuscate/Decode Files Or Information
+FYI,Auto-generated please review
+ARG,executable,C:\Windows\System32\calc.exe
```

Comment:
what is the point of defining ARG elements if unused?

Reply:
My observation was that the framework finds the ARG value from the criteria files first for interpolation. This allows us to specify our own ARG value from the criteria files without modifying any ARG values the yaml file from redcanary.

<img width="423" alt="image" src="https://github.com/secureworks/atomic-validation-criteria/assets/141030436/92629a94-f79c-428d-9ac0-e5d29a070393">

Reply:
yes, it gets used in the test.  However, I can't tell if the matching criteria takes the arg into account.  If I change the arg to something else, does it break the criteria matching?  If not, you can resolve this.

Reply:
If you are using the ARG value in the criteria and the ARG value is changed to some random value it will still get validated as the same ARG var is used in the test as well as criteria.